recipes-extended/xen/files/prune-vga-acpi-dev.patch

```diff
@@ -1,59 +0,0 @@
-################################################################################
```

Comment:
To be clear, removing this patch has no effect on the state of S3 for Windows 10?

Reply:
S3 isn't allowed on Windows 10 with or without this patch - the lack of S3 support for the graphics card is listed.

Run cmd as Administrator and run `powercfg /a`:
```
Standby (S3)
    An internal system component has disabled this standby state.
            Graphics
```

Reply:
Ack.  We will look into the S3 support issues after xen 4.16 gets merged.